src/compiler/emitter.ts

```diff
@@ -5662,11 +5662,13 @@ namespace ts {
                 detachedCommentsInfo = undefined;
             }
 
-            forEachLeadingCommentRange(currentSourceFile!.text, pos, cb, /*state*/ pos);
+            Debug.assertIsDefined(currentSourceFile);

```

Comment:
I would just exit early at the top of the function rather than assert.

Reply:
Done.